source/connect/tableau.txt

```diff
@@ -31,7 +31,7 @@ listening, and click :guilabel:`Sign In` to connect.
 
 By default, :program:`mongosqld` binds to port ``3307``.
 
-.. figure:: /images/bi-connector/tableau.png
```

Comment:
Believe it or not, the image needs to go here: https://github.com/mongodb/docs-assets/tree/bi-connector